### ISSUE TITLE
Dynamically Update Config

### ISSUE THREAD

**Issue by ldobson** [markparolisi/embedit-go#5]
Periodically reread config file and update config values. Make safe for use for multiple goroutines.

**Comment by markparolisi** [comment]
I'm not just what a good period would be. I could always pull the values fresh each time `utils.GetConfigValue` is called, but that's a bit wasteful.

**Comment by ldobson** [comment]
Agreed that a per-request refresh is wasteful. You could have a goroutine that just does a read every 10 seconds or so though, and updates the config either each time, or in the event of a change.
